src/services/FeedDataService.ts

```diff
@@ -31,6 +31,29 @@ export class FeedDataService {
     return {feeds: leavesAndFeeds};
   }
 
+  getParamsByFetcherName<T>(data: DeviationLeavesAndFeeds, fetcherName: string): T[] {
+    const {feeds} = data;
+
+    if (!feeds || Object.keys(feeds).length === 0) {
+      throw new Error('[FeedDataService] No leaves to process');
```

Comment:
why this is error? is it required to have feeds?

Reply:
Changed only to return an empty array